src/main/java/org/littleshoot/proxy/extras/SelfSignedSslEngineSource.java

```diff
@@ -150,8 +234,7 @@ public void checkServerTrusted(X509Certificate[] arg0,
             sslContext = SSLContext.getInstance(PROTOCOL);
             sslContext.init(keyManagers, trustManagers, null);
         } catch (final Exception e) {
-            throw new Error(
-                    "Failed to initialize the server-side SSLContext", e);
+            throw new Error("Failed to initialize the server-side SSLContext", e);
```

Comment:
Nevertheless the exceptions exist, so it is *possible*. I generally agree with @jekh that it's better to catch the specific exceptions, especially when editors will do that for you automatically these days.